test-data/unit/fixtures/typing-full.pyi

```diff
@@ -83,38 +93,52 @@ class AsyncGenerator(AsyncIterator[T], Generic[T, U]):
     @abstractmethod
     def __aiter__(self) -> 'AsyncGenerator[T, U]': pass
 
-class Awaitable(Generic[T]):
+@runtime
+class Awaitable(Protocol[T]):
     @abstractmethod
     def __await__(self) -> Generator[Any, Any, T]: pass
 
 class AwaitableGenerator(Generator[T, U, V], Awaitable[V], Generic[T, U, V, S]):
     pass
 
-class AsyncIterable(Generic[T]):
+@runtime
+class AsyncIterable(Protocol[T]):
```

Comment:
I just didn't want to touch ``Generator`` that is not a protocol and currently declared as invariant in the third type variable. I noticed that variance is mostly ignored in this file. If needed, I can make ``Generator``, ``Awaitable``, ``AsyncIterable``, and ``AsyncIterator`` covariant (these four can be only changed simultaneously). Should I do this?